src/etc/inc/system.inc

```diff
@@ -1653,6 +1653,16 @@ function system_ntp_configure() {
 	}
 	$ntpcfg .= "\n";
 
+	/* Add min-max ntp peers */
+	$ntpcfg .= "# Min-Max peers\n";
+	$ntpcfg .= 'tos minclock 3 maxclock ';
```

Comment:
There is already a line for `tos` which sets up orphan mode. Does this work with `tos` commands on two different lines, or does one override the other?

The best thing to do would be to combine them in one line, IMO.

Also, the default value for `minclock` in `ntpd` is already `3` so that can be omitted.

Reply:
ok i will work on it